cron/cron-daily.pl

```diff
@@ -241,7 +241,8 @@ sub watch_files {
           quoteHighBit => 1,
         );
         my $v = $d->stringify($File::Find::name);
-        warn sprintf qq{Found a bad directory v[%s], rmtree-ing}, $v;
+
```

Comment:
unnecessary blank line?

Reply:
Most blanks are, right? :)

Anyway, so much of the PAUSE code is without any vertical whitespace.  Sometimes I add some to help break things up.  This is maybe not the most obvious place to put it, but: I will continue to add vertical whitespace to create paragraphs of code.

Reply:
I like vertical whitespace, this just felt like a weird place to put it.